### ISSUE TITLE
JAWS doesn't announce gird column name only at the first column of a grid

### ISSUE THREAD

**Issue by Jan-Michael-Koensgen-SAP** [FreedomScientific/VFO-standards-support#550]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Open https://codepen.io/JAN-MICHAEL-KOENSGEN-SAP/pen/PomozRB in chrome
> 2. Start JAWS
> 3. Click "cell 1"

### Expected result

> "cell 1" together with column information "header 1" gets announced

### Actual result

> Only "cell 1" gets announced. Also using JAWS key + TAB to force it does not announce column information. 
> When doing the same at column "cell 2" as well as "cell 3" the corresponding column information gets announced.
> In VC PC cursor mode the column information also gets announced. 
> Setting tabindex at TD instead of the SPAN also leads to column announcement.

### Example

[>](https://codepen.io/JAN-MICHAEL-KOENSGEN-SAP/pen/PomozRB)

#### JAWS version and build number

2021.2105.53
 
#### Operating System and version
WIndows10

#### Browser and version:

Chrome 91.0.4472.106
